security/compliance_operator/co-scans/compliance-operator-supported-profiles.adoc

```diff
@@ -18,6 +18,6 @@ include::modules/compliance-supported-profiles.adoc[leveloffset=+1]
 [role="_additional-resources"]
 == Additional resources
 
-* xref:../../security/compliance_operator/compliance-operator-understanding.adoc#compliance_profiles_understanding-compliance[Compliance Operator profiles]
+* xref:../../../security/compliance_operator/co-scans/compliance-operator-supported-profiles.adoc#compliance_profiles_understanding-compliance[Compliance Operator profiles]
```

Comment:
This  link doesn't work at all. I think the ID string in the target file, compliance-profiles.adoc, is wrong:

> [id="compliance_profiles_{context}"]